x-pack/plugins/cases/server/connectors/cases/cases_connector_executor.test.ts

```diff
@@ -899,6 +900,206 @@ describe('CasesConnectorExecutor', () => {
+              errors: [],
+            });
+
+            mockBulkUpdateRecord.mockResolvedValue([{ ...oracleRecords[0], counter: 2 }]);
+
+            await connectorExecutor.execute({
+              ...params,
+              templateId: 'test_template_5',
+              reopenClosedCases: false,
+            });
+
+            expect(mockBulkUpdateRecord).toHaveBeenCalledWith([
+              { payload: { counter: 2 }, recordId: 'so-oracle-record-0', version: 'so-version-0' },
+            ]);
+
+            expect(casesClientMock.cases.bulkCreate.mock.calls[0][0]).toMatchInlineSnapshot(`
+              Object {
+                "cases": Array [
+                  Object {
+                    "assignees": Array [
+                      Object {
+                        "uid": "u_J41Oh6L9ki-Vo2tOogS8WRTENzhHurGtRc87NgEAlkc_0",
+                      },
+                    ],
+                    "category": "my category",
+                    "connector": Object {
+                      "fields": Object {
+                        "issueType": "Task",
+                        "parent": null,
+                        "priority": "Low",
+                      },
+                      "id": "jira-1",
+                      "name": "Jira",
+                      "type": ".jira",
+                    },
+                    "customFields": Array [
+                      Object {
+                        "key": "first_key",
+                        "type": "text",
+                        "value": "this is a text field value",
+                      },
+                    ],
+                    "description": "case desc",
+                    "id": "mock-id-4",
+                    "owner": "cases",
+                    "settings": Object {
+                      "syncAlerts": false,
+                    },
+                    "severity": "high",
+                    "tags": Array [
+                      "auto-generated",
+                      "rule:rule-test-id",
+                      "host.name",
+                      "host.name:A",
+                      "dest.ip",
+                      "dest.ip:0.0.0.1",
+                      "rule",
+                      "test",
+                      "sample-4",
+                    ],
+                    "title": "Case with sample template 5",
+                  },
+                ],
+              }
+            `);
+          });
+
+          it('creates a case with default values when template has required fields empty', async () => {
+            const newConfiguration = [
+              {
+                ...mockConfiguration[0],
+                customFields: [
+                  {
+                    ...mockConfiguration[0].customFields[0],
+                    defaultValue: null,
+                  },
+                ],
+                templates: [
+                  {
+                    key: 'test_template_5',
+                    name: 'Fifth test template',
```

Comment:
`name` and `description` are also `required` and can also be empty. Can we please test those too?

Reply:
These are template's `name` and `description`. Case's `name` and `description` are set to empty here by setting `caseFields: null`

Reply:
ah dammit, these always confuse me 😄